source/_components/hive.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: "Hive"
+description: "Instructions on how to integrate Hive devices with Home Assistant."
+date: 2017-09-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hive.png
+ha_category: Hub
+ha_release: 0.62
+ha_iot_class: "Cloud Polling"
+---
+
+
+This Hive component is the main component to setup and integrate all supported Hive devices.  Once configured with the minimum required details it will detect and add all your Hive devices in to Home Assistant, including support for multizone heating.
+
+This component uses the unofficial API used in the official Hive website [https://my.hivehome.com](https://my.hivehome.com) and you will need to use the same Username and Password you use on the Hive website to configure this Hive component in Home Assistant.
+
+To add all your Hive devices in to your Home Assistant installation add the following to your 'configuration.yaml' file:
+
+```yaml
+# Example configuration.yaml entry
+hive:
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+Configuration variables:
+
+- **username** (*Required*): Username from [https://my.hivehome.com](https://my.hivehome.com).
```

Comment:
Please use the configuration tags to list configuration variables. For more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
I don't understand what needs changing here.
Please could you point me to an existing file which demonstrates this in use and what I am missing?

Reply:
See for example this component: https://raw.githubusercontent.com/home-assistant/home-assistant.github.io/next/source/_components/device_tracker.hitron_coda.markdown

The configuration variables are listed using some code and then rendered. In your current version, they are simply markdown (which is the old way we did it).

Reply:
great stuff, thanks.

I have updated, but a little unsure of what to do with the option variable `scan_interval` and any examples needed with and without it, as the configuration tags documentation webpage said only add required variables.